include/bitcoin/protocol/zmq/message.hpp

```diff
@@ -66,12 +66,31 @@ class BCP_API message
 
     /// Remove a part from the queue top, false if empty queue or invalid.
     bool dequeue();
-    bool dequeue(uint32_t& value);
     bool dequeue(data_chunk& value);
     bool dequeue(std::string& value);
     bool dequeue(hash_digest& value);
     bool dequeue(address& value);
 
+    template<typename T,
```

Comment:
Avoid single letter symbols: `T`, suggest `Integer` or `Unsigned` (see` enqueue<Unsigned>`).

Reply:
`enqueue<Unsigned>` should be made consistent (type constraint and parameter name).

Reply:
Style: both templates should be listed before non-template methods.

Reply:
:+1: